front_end/ndb/NdbMain.js

```diff
@@ -185,17 +185,21 @@ Ndb.NodeProcessManager = class extends Common.Object {
     return this._processes.get(target.id()) || null;
   }
 
-  async addFileSystem(cwd) {
+  async addFileSystem(cwd, mainFileName) {
     let promise = this._cwds.get(cwd);
     if (!promise) {
       async function innerAdd() {
         const cwdUrl = Common.ParsedURL.platformPathToURL(cwd);
         const fileSystemManager = Persistence.isolatedFileSystemManager;
-        fileSystemManager.addPlatformFileSystem(cwdUrl, await Ndb.FileSystem.create(fileSystemManager, cwd, cwdUrl));
+        const fs = await Ndb.FileSystem.create(fileSystemManager, cwd, cwdUrl, mainFileName);
+        fileSystemManager.addPlatformFileSystem(cwdUrl, fs);
+        return fs;
       }
       promise = innerAdd();
       this._cwds.set(cwd, promise);
     }
+    if (mainFileName)
+      return (await promise).forceFileLoad(mainFileName);
```

Comment:
looks like you don't care about return value for this function?

Reply:
Yes, replaced return with await to return `undefined`.